swayfmt/src/utils/language/attribute.rs

```diff
@@ -40,41 +40,43 @@ impl Format for AttributeDecl {
         formatted_code: &mut FormattedCode,
         formatter: &mut Formatter,
     ) -> Result<(), FormatterError> {
-        let attr = self.attribute.get();
-        if attr.name.as_str() == "doc" {
-            if let Some(Some(doc_comment)) = attr
-                .args
-                .as_ref()
-                .map(|args| args.inner.final_value_opt.as_ref())
-            {
-                writeln!(formatted_code, "///{}", doc_comment.as_str().trim_end())?;
+        // `#`
+        write!(formatted_code, "{}", self.hash_token.span().as_str())?;
```

Comment:
The formatter check is failing here because the hash token is being written before checking if the attribute is a `doc` type